code/modules/reagents/newchem/drinks.dm

```diff
@@ -93,16 +94,32 @@
 
 /datum/reagent/ethanol/synthanol/on_mob_life(var/mob/living/M as mob, var/alien)
 
-	var/d = data 
-
-	if(d >= collapse_start && prob(10))
-		M.emote("collapse")
-		var/datum/effect/system/spark_spread/s = new /datum/effect/system/spark_spread
-		s.set_up(3, 1, M)
-		s.start()
-	if(d >= braindamage_start && prob(33))
-		M.adjustBrainLoss(1)
-	..()
+	var/d = data
```

Comment:
do NOT use data holy FUCK 

data is B A D

use the dedicated "how many times have I ticked" var holy FUCK BALLS SHIT AAAAAAAA